e2e/wdio/core/tests/notification.e2e-spec.ts

```diff
@@ -0,0 +1,144 @@
+import { NotificationPo } from '../pages/notification.po';
+import {
+    click,
+    getText,
+    refreshPage,
+    doesItExist,
+    getElementClass,
+    scrollIntoView,
+    waitForElDisplayed,
+    isElementDisplayed,
+    pause,
+    waitForNotDisplayed
+} from '../../driver/wdio';
+
+describe('Notification component test', function () {
+    const notificationPage = new NotificationPo();
+    const {
+        defaultExample,
+        openTemplateExample,
+        asContentExample,
+        groupExample,
+        button,
+        closeButton,
+        rejectButton,
+        approveButton,
+        tabsItem,
+        notificationBody,
+        notificationIndicator,
+        messageStrip,
+        result,
+        avatar,
+        notificationContainer,
+        cdkOverlay,
+        notification,
+        tabPanel,
+        notificationHeader,
+        overflowButton,
+        messageToast,
+        forwardButton,
+    } = notificationPage;
+
+    beforeAll(() => {
+        notificationPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForElDisplayed(notificationPage.title);
+    }, 2);
+
+    it('should check notification with avatar, success indicator and unread title', () => {
+        expect(isElementDisplayed(defaultExample + notificationBody + avatar)).toBe(true, `avatar does not exist`);
```

Comment:
could you please rework the selectors a bit. it's hard to read and debug when there are 3+ selectors being merged together.